receiver/jaegerreceiver/config.go

```diff
@@ -76,9 +76,9 @@ func DefaultServerConfigUDP() ServerConfigUDP {
 
 // Config defines configuration for Jaeger receiver.
 type Config struct {
-	config.ReceiverSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
-	Protocols               `mapstructure:"protocols"`
-	RemoteSampling          *RemoteSamplingConfig `mapstructure:"remote_sampling"`
+	*config.ReceiverSettings `mapstructure:"-"`
```

Comment:
It is not clear why this is embedded by pointer. It does not see to be optional, is it?

Reply:
Done. Fixed the pointer problem.